### PR TITLE
Implements isBetween plugin

### PR DESCRIPTION
**Pull request by lamartire**
Sets testURL in jest config for resolve issue with failing tests
#287

### CODE REVIEW

File: src/plugin/isBetween/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export default (o, c) => {
+  const proto = c.prototype
+  proto.isBetween = function (a, b) {
+    if (!(a instanceof c && b instanceof c)) {
+      throw new Error('isBetween arguments must be dayjs-like!')
+    }
+
```
Comment (iamkun) [inline src/plugin/isBetween/index.js]: `dayjs(a), dayjs(b)` might be better.

File: src/plugin/isBetween/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    return this.isAfter(a) && this.isBefore(b)
+  }
```
Comment (iamkun) [inline src/plugin/isBetween/index.js]: also do we need `this.isAfter(b) && this.isBefore(a)` ?

File: src/plugin/isBetween/index.js (hunk 1 above)
Comment (lamartire) [reply src/plugin/isBetween/index.js]: Of course.

File: src/plugin/isBetween/index.js (hunk 2 above)
Comment (lamartire) [reply src/plugin/isBetween/index.js]: What you mean? It's just re-using exist methods. Tell me more 🙂

Comment (iamkun) [reply src/plugin/isBetween/index.js]: Just not sure if we should check the two compared date in different order.

```
// the first one could be true, if we think it's useful
moment('2018-05-01').isBetween('2019-05-01', '2017-05-01') // false 
moment('2018-05-01').isBetween('2017-05-01', '2019-05-01') // true
```

Comment (lamartire) [reply src/plugin/isBetween/index.js]: Oh, I see. Give me few time.

**Comment by lamartire** [comment]
I changed logic, I think it is more readable and clear now.

File: src/plugin/isBetween/index.js (hunk 2 above)
Comment (iamkun) [reply src/plugin/isBetween/index.js]: see your latest commit, cool.

still, it's just a discussion of my idea not a decision

Comment (lamartire) [reply src/plugin/isBetween/index.js]: Why? If we working just with two dates it is really simple to compare our date with they. Or you mean more than 2 arguments?

Comment (iamkun) [reply src/plugin/isBetween/index.js]: no, just a discussion about if we should ignore the given order or not.

File: package.json [hunk 3]
```diff
@@ -26,6 +26,7 @@
       "test"
     ],
     "testRegex": "test/(.*?/)?.*test.js$",
+    "testURL": "http://localhost",
```
Comment (iamkun) [inline package.json]: what's this config used for?

Comment (lamartire) [reply package.json]: On osx last versions of jest throws error, `localStorage is not available for opaque origins`. It solves only with this option in config.

**Comment by lamartire** [comment]
@iamkun whats next? 🙂

**Comment by lamartire** [comment]
Wow! 🙌

**Pull request merged**